UI/UnoSQLiteOneDriveInvoiceSample/Demo/Demo/Demo.Shared/Pages/Settings.xaml

```diff
@@ -0,0 +1,192 @@
+﻿<Page
+    x:Class="Demo.Pages.Settings"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Demo.Pages"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:android ="http://uno.ui.android"
+    xmlns:ios="http://uno.ui.ios"
+    xmlns:macos="http://uno.ui/macos"
+    xmlns:wasm="http://uno.ui/wasm"
+    mc:Ignorable="d android ios macos wasm">
+
+    <ScrollViewer HorizontalScrollMode="Enabled" VerticalScrollMode="Enabled">
+        <Grid>
+            <android:Grid.RowDefinitions>
```

Comment:
Using AdaptiveTrigger would be better here, to allow the layout to adapt based on screen size rather than target platform

Reply:
Done